### DISCUSSION TITLE
Is Confluence 7.10 supported?

### DISCUSSION THREAD

**Discussion by david-drinn** [schorschii/companion-linux#6] [Q&A] [answered]
I see that the companion method introduced in Confluence 7.4 is supported. I followed those instructions, but when I click edit on a file in Confluence and xdg-open on the prompt, nothing happens. No error from the browser. No error in journalctl. Not sure where else to look.

**Comment by schorschii** [comment, marked as answer]
Yes, Confluence 7.10 is supported. And I just tested it with 7.12, this also works.

~~Can you please post the output of `xdg-open atlassian-companion://test` here?~~
I just saw that you got the "open with xdg-open" prompt in firefox, this means that the "atlassian-companion://" protocol scheme handler is installed correctly. Then I guess that the temporary directory `~/.cache/companion/tmp` does not exist. Strangely, I forgot to automatically create this directory in `companion2.py`, it was only implemented in the old `companion.py`. I added this now to the new script (ecb4843) and released a new version.

**Please try again with [v1.0.](https://github.com/schorschii/companion-linux/releases/tag/v1.0)**
